### ISSUE TITLE
Move current @ to a stylistic set and introduce a new 'normal' one

### ISSUE THREAD

**Issue by davelab6** [impallari/Cabin#25]
A user wrote to Google Fonts this week,

> First of all kudis to the designer. I love everything except this @ symbol in Cabin. This font is one of my favorite among Google fonts. Because of this @ design i can't use it for many websites that i love to use. I wish to change this @ type to a normal one. Kindly accept this as a request.

I've heard this a few times, so I'd like to request moving the current @ to a stylistic set.

**Comment by S-Simon** [comment]
Agreed, every time I show a design with Cabin to a client they are happy, except for the @ symbol.

**Comment by MarcMurrayUpfront** [comment]
Agreed, great font but I can't get the rationale behind the @ symbol.

**Comment by kenmcd** [comment]
It appears the alternate was already added as ss01.
From the fonts here in the repo:
![at_ss01](https://user-images.githubusercontent.com/844920/145293069-02f3b9d7-c006-4399-945f-6829f5c97c46.png)

**Comment by cstby** [comment]
Came to this repo to report this same issue. This font looks awesome, but the `@` symbol is almost a deal breaker.
